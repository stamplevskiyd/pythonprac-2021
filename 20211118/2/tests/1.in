class C:
        num = Num()
import string
print(C().num)
c, d = C(), C()
c.num = d.num = 2
print(c.num+d.num)
c.num = string.ascii_uppercase
print(c.num+d.num)
print(c.num)
