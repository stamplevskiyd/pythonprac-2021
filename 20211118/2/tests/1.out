0
4
28
26
